src/core/client/stream/tabs/Live/LiveComment/LiveCommentActionsContainer.tsx

```diff
@@ -188,7 +188,27 @@ const LiveCommentActionsContainer: FunctionComponent<Props> = ({
           isChat
           className={styles.action}
         />
-        {onReply && (
+        {comment.parent && onReply && (
```

Comment:
Line 226 has the same implementation of the `button` with just a different handler. It should be possible to combine the two.